special-pages/types.mjs

```diff
@@ -2,21 +2,33 @@
 import { join } from 'node:path';
 import { cwd, isLaunchFile } from '../scripts/script-utils.js';
 import { buildTypes } from '../types-generator/build-types.mjs';
+import { readdirSync } from 'fs';
```

Comment:
Is there a reason for these two separate imports or can they be rewritten as `import { existsSync, readdirSync } from 'node:fs’;`

Reply:
yep let me change it, just my editor being too quick for me again 😂